app/views/editor/modal/NewModelModal.coffee

```diff
@@ -22,6 +23,8 @@ module.exports = class NewModelModal extends ModalView
     model = new @modelClass
     name = @$el.find('#name').val()
     model.set('name', name)
+    if @modelClass.schema.default?
```

Comment:
The NewModelModal is used for several models. This `if` may apply to more than just the Level model, now or in the future. Instead of this check, use the className class property to explicitly add these default tasks only when the model is a Level. This should work:

```if @modelClass.className is 'Level'```